docs/Development/Optimizely.md

```diff
@@ -0,0 +1,13 @@
+[Optimizely.com](https://optimizely.com/) is a A/B testing tool that helps you optimize your site's conversion rate. In order to use this module, you'll need an Optimizely account. A Free 30 day trial account is available.
+
+From your account page at [https://app.optimizely.com/v2/accountsettings/account/overview](https://app.optimizely.com/v2/accountsettings/account/overview)
+Get the Account ID
+![Account ID field depicted](../../media/optimizely_account_id.png?raw=true)
```

Comment:
Images are not visible 
![image](https://cloud.githubusercontent.com/assets/1316234/23659600/49410a6c-034f-11e7-8622-b32d9b3f4241.png)

Let's call folder with images "assets" instead of "media".